pkg/runtimeconfig/defaults.go

```diff
@@ -75,3 +79,44 @@ func PathToK0sContainerdConfig() string {
 func PathToECConfig() string {
 	return "/etc/embedded-cluster/ec.yaml"
 }
+
+// ReplicatedAppDomain returns the replicated app domain. The first priority is the domain configured within the embedded cluster config.
+// The second priority is the domain configured within the license. If neither is configured, the default domain is returned.
+func ReplicatedAppDomain(license *kotsv1beta1.License) string {
```

Comment:
This does not return a domain it returns a url.  Should you rename the function?